doc/source/changes/version_0_25.rst.inc

```diff
@@ -213,6 +215,8 @@ Miscellaneous improvements
 
 * added icon to display in Windows start menu and editor windows.
 
+* freezed labels rows and columns in viewer.
```

Comment:
s/freezed/frozen/

unsure everybody would understand this. I'd write something like:
* the viewer keeps labels visible even when scrolling (label rows and columns are frozen).